### PR TITLE
Multi agency config

### PR DESCRIPTION
**Pull request by landonreed**
Update config for multi agency operation.

### CODE REVIEW

File: lib/config.yml [hunk 1]
```diff
@@ -26,6 +26,48 @@ routingTypes:
...
+    logo: https://d2tyb7byn1fef9.cloudfront.net/tram.png
+  - id: CTRAN
+    name: C-TRAN
+    logo: https://d2tyb7byn1fef9.cloudfront.net/ctran.png
+    order: 9999999
+  - id: '1' # accidentally forgot to add C-TRAN agency_id to routes.
+    name: C-TRAN
+    logo: https://d2tyb7byn1fef9.cloudfront.net/ctran.png
+    order: 9999999
+  - id: '2' # accidentally forgot to add C-TRAN agency_id to routes.
```
Comment (evansiroky) [inline lib/config.yml]: Why is C-TRAN in here 3 times?

**Comment by evansiroky** [comment]
This PR works in conjunction with https://github.com/opentripplanner/otp-react-redux/pull/62 which may need some more work. Assigning back to Landon until that is ready.

File: lib/config.yml (hunk 1 above)
Comment (landonreed) [reply lib/config.yml]: This was just to account for differences in GTFS between my local instance and the dev server. I think it should generally end up being `2` though, so I'll make that change.

Comment (landonreed) [reply lib/config.yml]: Actually, maybe we ought to just keep multiple for now so the deployment is not too fragile, I'll just add a clarifying comment.

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "parser": "babel-eslint"
   },
   "dependencies": {
+    "connected-react-router": "^5.0.1",
```
Comment (evansiroky) [inline package.json]: Let's remove this and bump to the latest otp-react-redux once https://github.com/opentripplanner/otp-react-redux/pull/66 is merged and released.

**Comment by landonreed** [comment]
@evansiroky, OK, I think I've resolved everything. Feel free to merge.

File: lib/main.js [hunk 3]
```diff
@@ -12,6 +12,8 @@ import ReactGA from 'react-ga'
 
 // import OTP-RR components
 import { createOtpReducer } from 'otp-react-redux'
+import { createHashHistory } from 'history'
+import { connectRouter, routerMiddleware } from 'connected-react-router'
```
Comment (evansiroky) [inline lib/main.js]: whoops, looks like we actually do need connected-react-router in the package.json for this library.

Comment (evansiroky) [reply lib/main.js]: history too

Comment (landonreed) [reply lib/main.js]: Fixed.

**Pull request merged**